### ISSUE TITLE
Not working in ubuntu 17.04 valet secure laravel

### ISSUE THREAD

**Issue by trixan** [cpriego/valet-linux#81]
Unit nginx.service has failed. 
php 7.1
```
 nginx: [emerg] BIO_new_file("/home/test/.valet/Certificates/laravel.app.crt") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/home/test/.valet/Certificates/laravel.app.crt'
```

**Comment by TheNodi** [comment]
Is the file really missing? Or is nginx having problems reading it?

Can you check the permissions on the folder to be sure valet can write file to it?

**Comment by trixan** [comment]
Yes, really missing.

**Comment by TheNodi** [comment]
Remove all the certificates and `~/.valet/Nginx/laravel.app` file, since valet doesn't behave well without the `.crt` file (unsecure doesn't work). Then try to re-secure the site to see if it works.

**Comment by cpriego** [comment]
@trixan I'm investigating this issue. It started suddenly and it breaks Valet.

**Comment by trixan** [comment]
without the .crt file is works, with re-secure the site - not works (((

**Comment by trixan** [comment]
@cpriego Thanks!

**Comment by cpriego** [comment]
Fixed in release [v2.0.20](https://github.com/cpriego/valet-linux/releases/tag/v2.0.20)

**Comment by trixan** [comment]
@cpriego Thanks, it works perfectly! :)
